qulice-checkstyle/src/test/resources/com/qulice/checkstyle/LocalVariableNames.java

```diff
@@ -35,6 +37,7 @@ void invalid() {
         try {
             int prolongations = 0;
             int camelCase = this.field;
+            int it = this.field;
             final int number1 = ++prolongations;
```

Comment:
@mkordas there`s a test at the valid method. this method is to test invalid names.